pkg/contexts/ocm/accessmethods/localfsblob/method.go

```diff
@@ -11,8 +11,10 @@ import (
 )
 
 // Type is the access type of a blob in a local filesystem.
-const Type = "localFilesystemBlob"
-const TypeV1 = Type + "/v1"
+const (
+	Type   = "localFilesystemBlob"
+	TypeV1 = Type + "/v1"
```

Comment:
If we change it anyway, could you use `TypeV1 = Type + runtime.VersionSeparator + "v1"` similar to the other occurrences.

Reply:
that was an auto-fixed change with `--fix`. In general if we have more than one `const` or `var` lines, convention to group them together with `const (...)` and `var (...)`, that's why the linter auto-fixed it.

Reply:
done.

Actions taken:
- applied suggested changes